app/services/reporting.js

```diff
@@ -127,13 +127,13 @@ export default Service.extend({
       return rhett;
     });
 
-    return resolve(mappedResults);
+    return mappedResults.filter(obj => isEmpty(year) || parseInt(obj.course.get('year')) === parseInt(year));
```

Comment:
please specify a radix of 10 as second argument when invoking `parseInt`. 
side-note: we should configure the eslinter to check for that (if that's an option).